generators/client-2/templates/_webpack-dev

```diff
@@ -0,0 +1 @@
+npm run webpack-dev
```

Comment:
@flaviencathala i think we can remove the file as its clear in the package.json and we now print the info during generation as well so thats sufficient